site-src/geps/gep-1058.md

```diff
@@ -0,0 +1,89 @@
+# GEP-1058: Route Inclusion and Delegation
+
+- Issue: [#1058](https://github.com/kubernetes-sigs/gateway-api/issues/1058)
+- Status: Provisional
+
+## TLDR
+
+Add support for a 2-tier hierarchy of route configuration to allow partial
+delegation of route settings and, provide more flexibility in how route
+configurations are organized and managed.
+
+## HTTPRoute Delegation Capabilities
+
+### Matching
+| Capability | Importance for Delegation | Complexity | Implementation Details |
+|---|---|---|---|
+| Hostnames | ? | Moderate | A three way intersection between Gateway, Parent Route, and Child Route hostnames |
+| Prefix Path on Parent, Prefix or Exact on Child | High | Low | Prefix path match of parent is prepended to child path match, child can support exact or prefix path matching |
+| Exact Path on Parent | Low | Moderate | Would require only "/" matches on child route |
+| Exact Path on Child | High | Low | Prefix path match of parent is prepended to child path match, child can support exact or prefix path matching |
+| Regex Path on Parent or Child | Low | Impossible | Maybe some complex merging is possible here, but I can't think of a good solution |
```

Comment:
What about delegating something like a `/accounts/[id]` pattern? This feels like it may be not a not-uncommon need - could something like excluding a parent PathPrefix make this possible? RegEx parent path feels like it could be harder and lower-value though, maybe split these cases apart?